packages/extension-base/src/services/chain-service/constants.ts

```diff
@@ -99,7 +100,8 @@ export const _STAKING_ERA_LENGTH_MAP: Record<string, number> = { // in hours
   creditcoin: 24,
   vara_network: 12,
   goldberg_testnet: 24,
-  manta_network: 6
+  manta_network: 6,
+  krest_network: 4
```

Comment:
where's the source for this?

Reply:
the block time is: 12s
block per round is: 1200 blocks (parachainStaking.defaultBlocksPerRound)
=> round_time = 12 * 1200 / 60 / 60 = 4 hours

More Infor:
number of unlocking blocks: 50400 blocks (parachainStaking.stakeDuration)
=> unlocking time = 50400*12/60/60/24 = 7 days